ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml

```diff
@@ -349,6 +349,11 @@ tests:
     - chain: ipi-openstack-pre
     - ref: fips-check
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-proxy
+  interval: 48h
+  steps:
+    cluster_profile: openstack-vexxhost
```

Comment:
shouldn't we run it on the same cloud as the presubmit?

Reply:
yeah I wanted to try how it works on vexxhost.